composer.json

```diff
@@ -3,19 +3,19 @@
     "description": "Shell for PHPCR",
     "require": {
         "php": "^5.6|^7.0",
-        "symfony/console": "~2.8|~3.0|~4.0",
+        "symfony/console": "~2.8|~3.0|~4.0|~5.0",
```

Comment:
can we use the `^` syntax? and i'd raise to `^3.4 | ^4.3 | ^5.0` to not have to deal with super old legacy. people stuck on that who don't use the shell stand alone won't care about the symfony 5 support anyways.

Reply:
just legacy, will update